pkgs/applications/virtualization/singularity/template.nix

```diff
@@ -0,0 +1,163 @@
+{ lib
+, util-linux
+, gpgme
+, openssl
+, libuuid
+, bash
+, coreutils
+, which
+, makeWrapper
+, cryptsetup
+, squashfsTools
+, nvidia-docker # For nvidia-container-cli
+, glibc # For ldconfig needed by nvidia-container-cli
+, go
+, buildGoModule
+, pname
+, version
+, src
+, projectName # "apptainer" or "singularity"
+  # These cannot be correctly override with `overrideAttrs`
+, vendorSha256 ? null
+, deleteVendor ? false
+, proxyVendor ? false
+, doCheck ? true
+, withoutSuid ? true # Required to run containers without suid
+, enableNvidiaContainerCli ? true
+}:
+
+with lib;
+let
+  propagatedBuildInputs = [ coreutils squashfsTools bash ];
```

Comment:
Does it really need `propagatedBuildInputs`? I think the absolute paths to `mksquashfs`&c get recorded in `singularity.conf`. I see `propagatedBuildInputs` were present in the previous expression, but maybe we can drop them?